crates/lox-time/src/subsecond.rs

```diff
@@ -189,7 +189,7 @@ mod tests {
     #[test]
     fn test_subsecond_into_f64() {
         let subsecond = Subsecond(0.0);
-        assert_eq!(0.0, subsecond.into());
+        assert_eq!(0.0, <Subsecond as Into<f64>>::into(subsecond));
```

Comment:
This certainly works! I suppose you could also do the following if the ugliness of the fully-qualified type annotations trouble you 😅 

```rust
        let subsecond = Subsecond(0.0);
        let as_f64: f64 = subsecond.into();
        assert_eq!(0.0, as_f64);
```

Reply:
Ah! This makes sense. I was only trying:

```
        assert_eq!(0.0f64, subsecond.into());
```

And was somehow expecting the type inference to work across the `assert_eq`, but it didn't work.